### ISSUE TITLE
regarding multiple testcase ids

### ISSUE THREAD

**Issue by hemanthsridhar** [allure-framework/allure-core#799]
Can I give multiple test case IDs. Because I am not able to using @TestCaseId("","",""). Just like how we give multiple Issues I feel it is better to have @TestCaseId as an array. Or if there is a way to do it, please help me out.

**Comment by baev** [comment]
Hi there!

Currently Allure doesn't support multiply test IDs. Could you please explain in more details why did you need such feature?

**Comment by volkovs** [comment]
Linking similar issue #773

**Comment by hemanthsridhar** [comment]
@baev  I need this feature because, sometimes I write a method which handles multiple test cases. Hence I need to give what test cases the particular method covers.

**Comment by thasherwin** [comment]
You can put your test case id in your data provider.
```
@TestCaseId("{0}")
    public void test(String testcase,String usernaame, String password){
}
```

**Comment by hemanthsridhar** [comment]
That's nice. Thank you :).

**Comment by hemanthsridhar** [comment]
But then again. I wish to have @Test case Id being multiple

**Comment by hemanthsridhar** [comment]
Any update on this?

**Comment by irushah** [comment]
Placeholder in TestCaseId does not work. It shows as String Test ID: {0} in the report. 

 I am looking for a similar feature. I have manual tests for each dataset and there is no way to tag each test case with these multiple testIds.

**Comment by hemanthsridhar** [comment]
@irushah  when you are doing the same solution as @thasherwin right? meaning reading from data provider

**Comment by irushah** [comment]
@hemanthsridhar  Yup, exactly, i am reading the first parameter from data provider 
Eg: For @TestCaseId("{0}") where first param is testcaseId = 123 below
```
@DataProvider(name = "dataset")
	private Object[][] getDataSet() {
		return new Object[][] {
				{ "123" , "username", "password"},
			};
}
```

**Comment by hemanthsridhar** [comment]
i am having the same problem too. @thasherwin

**Comment by thasherwin** [comment]
I checked also and it those not work indeed. I would expect this to work. Guess an issue or this was never implemented

**Comment by baev** [comment]
Closed as duplicate (see #773)
